packages/grafana-ui/src/components/Button/Button.tsx

```diff
@@ -311,3 +311,18 @@ export const clearButtonStyles = (theme: GrafanaTheme2) => {
     padding: 0;
   `;
 };
+
+export const clearLinkButtonStyles = (theme: GrafanaTheme2) => {
+  return css`
+    background: transparent;
+    border: none;
+    padding: 0;
+    font-family: inherit;
```

Comment:
Are `color` and `font-family` not already inherited attrs by default?

Reply:
Yes. But they are set with emotion on the Button element. The final css class generated for the button is going to look a little weird, it's going to have two entries for both color and font-family. The first one being a specific color/font (set by the styling in the button element) then these lines that gets tacked on and overrides those entries making the button inherit the font and color from the parent element.